app/views/wiki/show.html.erb

```diff
@@ -25,10 +25,11 @@
   <div class="alert alert-info" role="alert">This wiki page hasn't been edited for a while. If you see anything that needs updating, please help out!</div>
 <% end %>
 
-<% if (@wikis.nil? && @notes.nil?) || @node.has_tag('sidebar:featured') %>
-  <%= render :partial => "sidebar/featured" %>
-<% else %>
-  <%= render :partial => "sidebar/related" %>
+<% unless @node.has_tag('sidebar:none') %>
```

Comment:
Ah, one more thing. There are now too many `<% end %>` statements -- and forgive me if I miswrote the original issue, but I think we need to preserve this line just under the `sidebar:none` line:

```ruby
-<% if (@wikis.nil? && @notes.nil?) || @node.has_tag('sidebar:featured') %>
```

If you add that back in, the # of `<% end %>`s will be correct and the tests should pass! 

Thanks for your persistence on this! It's complex stuff!

Reply:
Ah - that's my fault. OK, fingers crossed all good here.